index.html

```diff
@@ -0,0 +1,283 @@
+			input.link {border:2px solid #d9c7b4;}
+			input.link::placeholder {color:#d9c7b4;}
+			input.link:focus {border:2px solid blue;}
+			input.link:focus::placeholder {color:transparent;}
+			input.submit {font-weight:800;border:2px solid #d9c7b4;background-color:#d9c7b4;color:white;}
+			input.submit:hover {cursor:pointer;}
+			input.submit:disabled {opacity:0.25;cursor:not-allowed;}
+			table,td,th,tbody{display:block!important;text-align:center;}
+			td,th{padding-right:1vw;}
+			span,i,b{display:inline!important;}
+			a{text-decoration:underline;color:blue;display:inline!important;width:auto!important;}
+		</style>
+	</head>
+	<body>
+		<main>
+			<section class="intro row">
+				<div class="intro_info col flex-60 pad-r">
+					<div class="row pad-r h-big item-center">
+						<h1 class="blue strong">Link Biscuit</h1>
+					</div>
+					<div class="row pad-r h-norm item-center">
+						<span class="dimgray">&#9654; Use this place for storing interesting pieces of the web you <b>actually</b> want to read.</span>
+					</div>
+				</div>
+				<div class="intro_input col flex-40">
+					<form class="col-rev flex-100">
+					<div class="row h-norm item-center">
+						<div class="col flex-75 pad-r item-center">
+							<span id="help" class="green"></span>
+						</div>
+						<div class="col flex-25 item-center self-center">
+							<input class="submit h-norm content-center strong" id="submit" type="submit" value="Add link" disabled>
+						</div>
+					</div>
+					<div class="row h-big item-center">
+						<input class="link h-norm blue pad-r pad-l" id="link" type="text" autocomplete="off" spellcheck="false" autocorrect="off" required placeholder="https://medium.com/swlh/sexism-in-tech-an-inconvenient-truth-26df0329e39">
+					</div>
+					</form>
+				</div>
+			</section>
+			<section id="listing" class="listing">
+				<table>
+					<thead>
+						<tr>
+							<th class="flex-7-5 strong silver">State</th>
+							<th class="flex-80 left strong silver">Link</th>
+							<th class="flex-12-5 strong silver">Added</th>
+						</tr>
+					</thead>
+					<tbody>
+					</tbody>
+				</table>
+
+			</section>
+		</main>
+
+	</body>
+	<script type="text/javascript">
+		// URL validation function, links that pass and don't pass listed here https://stackoverflow.com/a/15855457
+		// FAILS ON 'https://www.google'
+		let validateLink = function (value) {
+		  return /^(?:(?:(?:https?|ftp):)?\/\/)(?:\S+(?::\S*)?@)?(?:(?!(?:10|127)(?:\.\d{1,3}){3})(?!(?:169\.254|192\.168)(?:\.\d{1,3}){2})(?!172\.(?:1[6-9]|2\d|3[0-1])(?:\.\d{1,3}){2})(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*(?:\.(?:[a-z\u00a1-\uffff]{2,})))(?::\d{2,5})?(?:[/?#]\S*)?$/i.test(value);
+		}
+
+ 		// Parsing date function; NOT READY FOR DIFFERENT TIMEZONES
+		let parseDate = function (input) {

```

Comment:
I'd prefer `const` over `let` here cause the function is immutable. Use `let` if the value should change eventually.